README.md

```diff
@@ -48,3 +40,19 @@ To runs tests use the command `npm test`.
 
 ## Project Bootstrap
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+
+## API Calls
```

Comment:
we should also add GET request for `api/reset/: token` to request a page which allows user to enter a new password.